src/pages/Auth/Signup/Signup.jsx

```diff
@@ -86,6 +89,7 @@ export const Signup = () => {
           Create Password
           <input
             type="password"
+            autocomplete="on"
```

Comment:
Useful attribute!

Reply:
yes its giving me warning about that thing so added